### PR TITLE
Release notes for 0.23.10

### PR DESCRIPTION
**Pull request by HeroicKatora**
Sometimes in the next few days (potentially integrating `tiff` but also, not depending on it).

### CODE REVIEW

**Comment by fintelia** [comment]
There's some references to the stable compiler. Do those really not work if you compile with nightly? I thought anything that worked with stable was supposed to work with subsequent nightlies, but perhaps I'm misunderstanding?

**Comment by HeroicKatora** [comment]
They don't work with `1.34.2` unlike the other features and the MSRV promise.

**Comment by fintelia** [comment]
Perhaps "requires _latest_ stable rustc"?

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ Rust image aims to be a pure-Rust implementation of various popular image format
 
 ## Changes
 
+### Version 0.23.10
+
+- Added AVIF encoding capabilities using the `ravif` crate. Please note that
+  the feature targets the stable compiler and is not enabled by default.
```
Comment (fintelia) [inline CHANGES.md:16]: stable -> latest stable

**Review by fintelia — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (HeroicKatora) [reply CHANGES.md:16]: Ah yes, forgot that I mentioned that here as well :+1:

**Pull request merged**